test/unit/specs/components/related-images.spec.js

```diff
@@ -1,14 +1,45 @@
-import RelatedImage from '~/components/RelatedImages'
+import RelatedImages from '~/components/ImageDetails/RelatedImages'
```

Comment:
Would it be possible to conver this test suite to use `testing-library`? We're re-writing it already 🙂